### PR TITLE
Explain different attestations and add missing info for kwctl

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Relates to https://github.com/kubewarden/kubewarden-controller/issues/856.

## Test

Tested commands locally by hand.

### CODE REVIEW

File: docs/tutorials/verifying-kubewarden.md [hunk 1]
```diff
@@ -98,21 +105,21 @@ $ cosign verify-blob --bundle audit-scanner-attestation-amd64-checksum-cosign.bu
...
 
 ## Helm charts
 
 You can find our Helm charts in our `https://` traditional Helm repository under
 https://charts.kubewarden.io.
 
 The same Helm charts are signed via Sigstore's keyless signing, and pushed to an
-OCI repository that can hold both the Helm chart and its signature as OCI
-artifacts.
+OCI repository that can hold both the Helm chart, its signature, and its
```
Comment (jhkrug) [inline docs/tutorials/verifying-kubewarden.md:121]:
```suggestion
OCI repository that holds both the Helm chart, its signature, and its
```

File: docs/tutorials/verifying-kubewarden.md [hunk 2]
```diff
@@ -189,10 +200,18 @@ Verified OK
 You can then verify that the cert in the returned json contains the correct
 issuer, subject, and `github_workflow_repository` extensions.
 
+The SBOMs are signed and published in the [GitHub
+Release](https://github.com/kubewarden/kwctl/releases) of the project.
+
+The provenance attestation for `kwctl` can be verified by using [`gh
```
Comment (jhkrug) [inline docs/tutorials/verifying-kubewarden.md:206]:
```suggestion
The provenance attestation for `kwctl` is verified by using [`gh
```

**Review by jhkrug — APPROVED** [review]
Two tiny suggestions if you like them. Otherwise lgtm

**Comment by viccuad** [comment]
Thanks for the review!

**Pull request merged**